### PR TITLE
Map conflated components to an existing component

### PR DESCRIPTION
**Pull request by marco-c**
And some minor fixes.

### CODE REVIEW

File: auto_nag/scripts/component.py [hunk 1]
```diff
@@ -62,6 +74,9 @@ def get_bugs(self, date='today', bug_ids=[]):
         # Retrieve history.
         self.retrieve_history(bugs)
 
+        # Remove bugs for which somebody has modified the product/component.
+        self.remove_using_history(bugs)
```
Comment (calixteman) [inline auto_nag/scripts/component.py]: We can probably avoid that in improving bz query (and so less bugs to retrieve):
not (f1='product', o1='changedafter', v1=start_date) and not(f2='component', o2='changedafter', v2=start_date))
and by the way, you can use (f=creation_ts, o=greaterthan, v=start_date) to get bugs created after start_date

Comment (marco-c) [reply auto_nag/scripts/component.py]: Done with 4be84d9c5db8d74226695ec68abf9a73ea2e92d8.

File: auto_nag/scripts/component.py [hunk 2]
```diff
@@ -49,22 +50,13 @@ def get_bz_params(self, date):
             'chfield': '[Bug creation]',
             'chfieldfrom': start_date,
             'chfieldto': end_date,
+            # Ignore bugs for which somebody has ever modified the product or the component.
+            'n1': 1, 'f1': 'product', 'o1': 'changedafter', 'v1': datetime(1970, 1, 1),
```
Comment (calixteman) [inline auto_nag/scripts/component.py]: nit: you can remove datetime and just use '1970-01-01'

Comment (marco-c) [reply auto_nag/scripts/component.py]: Done with de2ed31.

**Pull request merged**